todoapp/src/components/Todos.vue

```diff
@@ -1,26 +1,37 @@
 <!-- eslint-disable vue/multi-word-component-names -->
 <script setup lang="ts">
 import TodoUl from './TodoUl.vue'
-import type { TextString } from '../types/text'
+import NoTodosMobile from '../assets/notodosmobile.svg'
+import NoTodos from '../assets/notodos.svg'
+import type { TodoTypes } from '../types/text'
 
 const props = defineProps<{
-  todos: TextString[]
+  todos: TodoTypes[]
+  isMobile: boolean
 }>()
-const emits = defineEmits<{
-  (e: 'removeTodo', index: number): void
-}>()
-
-function removeTodo(index: number) {
-  emits('removeTodo', index)
-}
 </script>
 
 <template>
   <div class="todos">
-    <p v-if="todos.length > 0">These are your tasks:</p>
-    <p v-else>You have no tasks</p>
-    <TodoUl :todos="todos" @remove-todo="removeTodo" />
+    <TodoUl :is-mobile="isMobile" :todos="todos" />
+    <img :src="isMobile ? NoTodosMobile : NoTodos" v-if="todos.length < 1" />
```

Comment:
The svg for no todo is the same for desktop and mobil so please use only one svg and create a separate vue component for this empty state which has an img tag with the svg and a p tag for the text below and the only difference between the mobile and desktop version will be the size of image and the font size